wrangler.toml

```diff
@@ -1,6 +1,7 @@
 name = "ubiquibot-worker"
 main = "src/worker.ts"
 compatibility_date = "2023-12-06"
+node_compat = true
```

Comment:
Hey @gitcoindev I don't think we can add this 

@whilefoo rfc

Reply:
@pavlovcik @whilefoo I added this config to silence cloudflare worker warning. I decided to dig into this deeper and discovered that this warning may be related to a bug https://github.com/cloudflare/workers-sdk/issues/4050 which was coincidentally put to `in progress` 40 minutes ago. Let's monitor this and remove once it will be fixed.

Reply:
But also doesn't our npm library depend on node APIs at runtime? If so we can't actually use it in our production Worker.

Reply:
ubiquity-logger uses probot's context so that needs to be modified, it uses supabase-js which needs some configuration to work on CF workers (https://github.com/supabase/supabase-js?tab=readme-ov-file#custom-fetch-implementation) and also it uses node's util package